pvlib/tests/test_mlfm.py

```diff
@@ -0,0 +1,262 @@
+
+import numpy as np
+import pandas as pd
+from pvlib import mlfm
+from numpy.testing import assert_allclose  # assert_almost_equal,
```

Comment:
Remove commented out import.

Reply:
Also, imports are typically alphabetized and broken out into standard library packages/modules, external packages/modules, and internal packages/modules.